### PR TITLE
create demo for f:ajax and tc:event

### PR DESCRIPTION
**Pull request by JasminKroeger**
* this example show how to use f:ajax and tc:event together inside a button
* the ajax event is called with a click and the event with a double click

Issue: TOBAGO-1695

### CODE REVIEW

File: tobago-example/tobago-example-demo/src/main/java/org/apache/myfaces/tobago/example/demo/BehaviorController.java [hunk 1]
```diff
@@ -37,6 +37,8 @@
 
   private String ajax;
   private String event;
+  private String output;
+  private String selected;
```
Comment (henningn) [inline tobago-example/tobago-example-demo/src/main/java/org/apache/myfaces/tobago/example/demo/BehaviorController.java:41]: selected is not used

File: tobago-example/tobago-example-demo/src/main/java/org/apache/myfaces/tobago/example/demo/BehaviorController.java [hunk 2]
```diff
@@ -59,6 +61,22 @@ public int getCounter() {
...
 
+  public String getOutput() {
+    return output;
+  }
+
+  public void setOutput(String output) {
+    this.output = output;
+  }
+
+  public String getSelected() {
```
Comment (henningn) [inline tobago-example/tobago-example-demo/src/main/java/org/apache/myfaces/tobago/example/demo/BehaviorController.java:72]: selected is not used; getter/setter can be removed

File: tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml [hunk 3]
```diff
@@ -96,5 +96,23 @@
...
+  &lt;f:ajax event="click" render="out" listener="\#{behaviorController.eventOutput}"/>
+  &lt;tc:event event="dblclick" actionListener="\#{behaviorController.eventOutput}"/>
+&lt;/tc:button>
+&lt;tc:out id="out" value="Output: \#{behaviorController.output}"/></code></pre>
+      <tc:segmentLayout large="6seg 6seg">
+        <tc:button id="btnAjaxEvent" label="Submit" type="submit">
+          <f:ajax event="click" render="out" listener="#{behaviorController.eventOutput}"/>
+          <tc:event event="dblclick" actionListener="#{behaviorController.eventOutput}"/>
+        </tc:button>
+        <tc:out id="out" value="Output: #{behaviorController.output}"/>
```
Comment (henningn) [inline tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml:114]: Use the label attribute for "Output" as in the other examples: <tc:out id="out" label="Output" value="#{behaviorController.output}"/>

File: tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml [hunk 4]
```diff
@@ -96,5 +96,23 @@
...
+      <p>This is an example to show how <code class="language-markup">&lt;f:ajax/></code> and <code
+              class="language-markup">&lt;tc:event/></code> can be used together.
+        The ajax event is called with a click and the event with a double click.</p>
+      <pre><code class="language-markup">&lt;tc:button id="btnAjaxEvent" label="Submit" type="submit">
+  &lt;f:ajax event="click" render="out" listener="\#{behaviorController.eventOutput}"/>
+  &lt;tc:event event="dblclick" actionListener="\#{behaviorController.eventOutput}"/>
+&lt;/tc:button>
+&lt;tc:out id="out" value="Output: \#{behaviorController.output}"/></code></pre>
+      <tc:segmentLayout large="6seg 6seg">
+        <tc:button id="btnAjaxEvent" label="Submit" type="submit">
```
Comment (henningn) [inline tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml:110]: type="submit" is for HTML button tags. In Tobago this is unnecessary.

File: tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml [hunk 5]
```diff
@@ -96,5 +96,23 @@
...
+          <f:ajax event="click" render="out" listener="#{behaviorController.eventOutput}"/>
```
Comment (henningn) [inline tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml:111]: "click" is the default event type. So you might remove the event attribute.

File: tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml [hunk 6]
```diff
@@ -96,5 +96,23 @@
...
+    <tc:section label="f:ajax and tc:event">
+      <p>This is an example to show how <code class="language-markup">&lt;f:ajax/></code> and <code
+              class="language-markup">&lt;tc:event/></code> can be used together.
+        The ajax event is called with a click and the event with a double click.</p>
+      <pre><code class="language-markup">&lt;tc:button id="btnAjaxEvent" label="Submit" type="submit">
+  &lt;f:ajax event="click" render="out" listener="\#{behaviorController.eventOutput}"/>
+  &lt;tc:event event="dblclick" actionListener="\#{behaviorController.eventOutput}"/>
+&lt;/tc:button>
+&lt;tc:out id="out" value="Output: \#{behaviorController.output}"/></code></pre>
+      <tc:segmentLayout large="6seg 6seg">
```
Comment (henningn) [inline tobago-example/tobago-example-demo/src/main/webapp/content/30-concept/30-behavior/Behavior.xhtml:109]: If you use the label layout of the tc:out, the layouting looks good for me. So you might remove the segmentLayout. But if you want to use the segmentLayout, the second "6seg" is unnecessary. You can write: <tc:segmentLayout large="6seg">

**Comment by JasminKroeger** [comment]
I pushed the changes

**Pull request merged**